src/context.rs

```diff
@@ -814,7 +814,18 @@ impl Context {
     pub async fn get_next_msgs(&self) -> Result<Vec<MsgId>> {
         let last_msg_id = match self.get_config(Config::LastMsgId).await? {
             Some(s) => MsgId::new(s.parse()?),
-            None => MsgId::new_unset(),
+            None => {
+                self.sql
+                    .query_row(
+                        "SELECT IFNULL((SELECT MAX(id) - 1 FROM msgs), 0)",
```

Comment:
Why `- 1`?

Reply:
Probably `Config::LastMsgId` should be renamed to `LastProcessedMsgId`, and `last_msg_id` here too accordingly, then maybe this question wouldn't arise, but not sure because i already read the code below